mslice/app/mainwindow.ui

```diff
@@ -250,7 +250,7 @@
                   </property>
                   <property name="maximumSize">
                    <size>
-                    <width>110</width>
+                    <width>130</width>
```

Comment:
Maybe we can reset the widths of the "Rename" and "Overplot" buttons to 130 - at the moment it looks a bit odd with only this button being wider...

e.g.

```diff
diff --git a/mslice/app/mainwindow.ui b/mslice/app/mainwindow.ui
index e43d6129..071de71f 100644
--- a/mslice/app/mainwindow.ui
+++ b/mslice/app/mainwindow.ui
@@ -75,7 +75,7 @@
                 <widget class="QPushButton" name="btnRename">
                  <property name="maximumSize">
                   <size>
-                   <width>110</width>
+                   <width>130</width>
                    <height>16777215</height>
                   </size>
                  </property>
@@ -166,7 +166,7 @@
                 <widget class="QPushButton" name="btnOverplot">
                  <property name="maximumSize">
                   <size>
-                   <width>110</width>
+                   <width>130</width>
                    <height>16777215</height>
                   </size>
                  </property>
```

Reply:
Subtract and Compose also need to change since Rename is also on the 2D tab. I've now just set maximumSize for width to 130 for all buttons.